src/pages/notebook.tsx

```diff
@@ -230,19 +241,35 @@ export const notebookPlugin: JupyterFrontEndPlugin<void> = {
     const sidebarItem = new SidebarIcon({
       label: 'Notebook',
       icon: EverywhereIcons.notebook,
+      pathName: `${(router?.base || '').replace(/\/$/, '')}/lab/index.html`,
       execute: () => {
         if (readonlyTracker.currentWidget) {
-          return shell.activateById(readonlyTracker.currentWidget.id);
+          const id = readonlyTracker.currentWidget.id;
+          shell.activateById(id);
+          return SidebarIcon.delegateNavigation;
         }
         if (tracker.currentWidget) {
-          return shell.activateById(tracker.currentWidget.id);
+          const id = tracker.currentWidget.id;
+          shell.activateById(id);
+          return SidebarIcon.delegateNavigation;
```

Comment:
I like the idea, but this currently break marking sidebar items as active because the truthy return value so far had "cancel" semantics:

https://github.com/JupyterEverywhere/jupyterlite-extension/blob/970bc3d4dde4797afc5ce5478077d5a6aed3b83d/src/sidebar.ts#L45-L47

The code in `sidebar.ts` would need to be changed accordingly.

![Image](https://github.com/user-attachments/assets/0ed6f06f-f282-424a-ad43-52bd1bbbcc79)

Reply:
Thanks, good catch!

Reply:
Actually, I am unable to reproduce this on my end, but could you check if c2e84d6d362f48fbd739caf7fbf137181b39b5f5 that I just pushed resolves this?

Reply:
Probably; I'll open an issue if I keep seeing it. Thanks for checking!